diff --git a/chrome/chrome_browser.gypi b/chrome/chrome_browser.gypi
index 4679e50..97b6402 100644
--- a/chrome/chrome_browser.gypi
+++ b/chrome/chrome_browser.gypi
@@ -3280,7 +3280,7 @@
             '../build/linux/system.gyp:gnome_keyring',
           ],
         }],
-        ['OS=="linux" and chromeos!=1', {
+        ['OS=="linux" and chromeos!=1 and <(ozone_platform_wayland) != 1', {
           'sources': [ '<@(chrome_browser_libsecret_sources)' ],
           'defines': [
             'USE_LIBSECRET',
